<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetConf</class>
 <widget class="QWidget" name="widgetConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,3,0,1,0">
   <item>
    <widget class="QGroupBox" name="groupBaseInfo">
     <property name="title">
      <string>基本信息</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_33" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>工程名称</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditProjectName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_34" stretch="0,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>项目名称</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditItemName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="1,20">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>项目类型</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboItemType">
          <item>
           <property name="text">
            <string>成孔检测</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>成槽检测</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupConf">
     <property name="title">
      <string>配置参数</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>设计孔深</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditHoleDepth">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>设计直径</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditHoleDiameter">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>估计声速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditSpeed">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>零声时</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditZeroSound">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_28">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>编码间距</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditDistance">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_29">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>测点间距</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lEditPointDistance">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_30">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>探头直径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditDiameter">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>串口</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>数据串口</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>串口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboPortData">
             <item>
              <property name="text">
               <string>com1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBaudData">
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>230400</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>高度串口</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>串口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboPortHeight">
             <item>
              <property name="text">
               <string>com1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>com3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttySAC2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBaudHeight">
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupFileInfo">
     <property name="title">
      <string>文件信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>文件名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEditFileName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
